picr22-enne-1-voistlust/testing.py

```diff
@@ -0,0 +1,14 @@
+import struct
+import serial
+import time
+import MainMovement as MP
+
```

Comment:
Are you using this file?

Reply:
didnt use any files in picr22-enne-1-voistlust so removed it